### DISCUSSION TITLE
Github co-pilot access for accounts with student developer pack

### DISCUSSION THREAD

**Discussion by epicman25** [community/community#23105] [New to GitHub]
<p>I have github student developer pack enabled on my account.According to the website verified students will have access to github copilot.But when I checked I didn’t have access to github copilot.So what should I do now?</p>

**Comment by queenofcorgis** [comment, marked as answer]
<p>Howdy :cowboy_hat_face:</p>
<p>Have you checked out the thread <a class="mention" href="/yistarostin">@yistarostin</a> posted <a href="https://github.community/t/how-can-students-get-access-to-copilot-through-student-pack/257736/3">here</a>? They said it was able to help!</p>
